src/components/MainContainer/MainContainer.jsx

```diff
@@ -2,17 +2,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './MainContainer.module.css';
 import FilmsContainer from '../FilmsContainer/';
+import { connect } from 'react-redux';
 
-function MainContainer(props) {
-    return (
-        <main className={styles.mainContainer}>
-            {props.films ? <FilmsContainer films={props.films} /> : null}
-        </main>
-    );
+class MainContainer extends React.Component {
```

Comment:
Why you use the class component for just view. Could you change it to the functional component?

Reply:
Fixed